ImperatorToCK3/Source/Imperator/Characters/Character.cpp

```diff
@@ -75,12 +75,12 @@ void Imperator::Character::registerKeys()
 	registerKeyword("spouse", [this](const std::string& unused, std::istream& theStream) {
 		const commonItems::ullongList spouseList(theStream);
 		for (const auto spouse : spouseList.getULlongs())
-			spouses.insert(std::pair(spouse, nullptr));
+			spouses.emplace(spouse, nullptr);
```

Comment:
It's not a big deal either way, but https://abseil.io/tips/112 suggests insert over emplace

Reply:
I chose it because of how cleaner it looks, and emplace seems to work correctly with the types in this map.

Reply:
I could use operator[] too.